webpack.config.js

```diff
@@ -7,13 +7,26 @@ module.exports = {
     'gridstack-static': './src/gridstack-static.ts'
   },
   mode: 'production', // production vs development
-  devtool: 'source-map',
-  // devtool: 'eval-source-map', // for best (large .js) debugging. see https://survivejs.com/webpack/building/source-maps/
+  // devtool: 'source-map',
+  devtool: 'eval-source-map', // for best (large .js) debugging. see https://survivejs.com/webpack/building/source-maps/
+  target: ['web', 'es5'],
   module: {
     rules: [
       {
         test: /\.ts$/,
-        use: 'ts-loader',
+        use: [
+          {
+            loader: 'babel-loader',

```

Comment:
no babel please, already have webpack and ts compiler... don't think we need that many packages!